packages/template-retail-react-app/app/commerce-api/hooks/useShopper.js

```diff
@@ -41,7 +41,7 @@ const useShopper = (opts = {}) => {
             customer.isInitialized &&
             customer.customerId !== basket.customerInfo.customerId
         ) {
-            basket.getOrCreateBasket()
+            basket.getBasket()
             return
```

Comment:
These two block does not make sense anymore since getBasket only does one thing which is to fetch basket. Previously it did two things create or get basket. Now it only gets basket. Can we find a better logic here?

Reply:
Same as above.
Reverted useBasket back to original code.